
#Array of:
int8 ID
float64 xPos
float64 yPos
float64 hdg
